platform/broadcom/sonic-platform-modules-dell/s5232f/sonic_platform/chassis.py

```diff
@@ -38,7 +38,7 @@ class Chassis(ChassisBase):
 
     oir_fd = -1
     epoll = -1
-
+    REBOOT_CAUSE_PATH = "/host/reboot-cause/platform/reboot_reason"
```

Comment:
If this line is necessary, I think this line should also be there in s5248/sonic_platform/chassis.py.

Reply:
This line already present in s5248f. https://github.com/Azure/sonic-buildimage/blob/acfee3be9a732bc3f2532dcf33f10d9dd885a71c/platform/broadcom/sonic-platform-modules-dell/s5248f/sonic_platform/chassis.py#L57